helper.php

```diff
@@ -43,6 +43,18 @@ public function areWriteable($files) {
         return true;
     }
 
+    public function actuallySaveFile($file, $content) {
```

Comment:
This function is already called from `saveToFile()` twice and from `save()` once. So the intention was to emphasize that this function is actually saving the file. However since I am now switching to `io_saveFile()` and hence calling another "save" function, I will rename it.